Sources/SwifQL/SwifQL.swift

```diff
@@ -67,5 +67,5 @@ extension Array where Element == SwifQLable {
 
 struct FQP {
     static var key = "§§§"
-    static func fKey(_ i: Int) -> String { return "$\(i)" }
+    static func fKey(_ i: Int) -> String { return "?" }
```

Comment:
As far as I know we need `$1 $2 $3` for binded variables, at least in Vapor. If we can use just `?` instead then this function make no sense anymore and it should be deleted or replaced with just `static var fKey = "?"`.

Reply:
using $1 mysql can't bind params and give error

Reply:
Ok, then this change will brake `PostgreSQL`. Then somewhere there should be dialect check 🙂

Reply:
true - i didn't checked `PostgreSQL`

Reply:
maybe save dialect from prepare for future use ?

Reply:
Yeah `SwifQLPrepared` struct should have dialect inside and then in its `splitted` calculated variable check for dialect should be added. 🙂

Reply:
I've just implemented `fKey` for both dialects and will push that change by myself, could you please remove this change from this pull request to avoid merge conflicts.